### DISCUSSION TITLE
Are there sync versions of methods: youtube.Videos.ClosedCaptions.GetAsync and youtube.Videos.ClosedCaptions.GetManifestAsync

### DISCUSSION THREAD

**Discussion by lofti198** [Tyrrrz/YoutubeExplode#778] [Q&A]
Using from legacy sync code and need sync methods ideally instead of those:
youtube.Videos.ClosedCaptions.GetAsync and youtube.Videos.ClosedCaptions.GetManifestAsync

**Comment by Sekoree** [comment]
Probably wouldn't make much sense to add those since both have underlying async calls.
Maybe just call them synchronously yourself? There's all sorts of ways to do that

**Reply by lofti198** [reply]
I am actually doing this, but somewhy in different environments using Task.Run returning .Result for example return either expected result or null ref exception)))
